src/Employer/Employer.Web/Orchestrators/Part1/TitleOrchestrator.cs

```diff
@@ -71,9 +78,23 @@ public TitleViewModel GetTitleViewModel()
 
             vacancy.Title = vm.Title;
 
-            await _client.UpdateVacancyAsync(vacancy);
+            try
```

Comment:
Similar comment to that above. Can we move the below boilerplate code into the base class?

Reply:
👍